cpp/open3d/core/MemoryManager.cpp

```diff
@@ -114,7 +114,9 @@ std::shared_ptr<DeviceMemoryManager> MemoryManager::GetDeviceMemoryManager(
     if (map_device_type_to_memory_manager.find(device.GetType()) ==
         map_device_type_to_memory_manager.end()) {
         utility::LogError(
-                "MemoryManager::GetDeviceMemoryManager: Unimplemented device");
+                "MemoryManager::GetDeviceMemoryManager: Unimplemented device "
```

Comment:
I think `LogError` prints the function name by default, so `MemoryManager/GetDeviceMemoryManager` will be printed by default. So may skip `MemoryManager::GetDeviceMemoryManager:`.

Reply:
Done.